include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -3100,12 +3100,12 @@ __parallel_set_op(_ExecutionPolicy&& __exec, _ForwardIterator1 __first1, _Forwar
 
     return __internal::__except_handler([&__exec, __n1, __first1, __last1, __first2, __last2, __result, __is_vector,
                                          __comp, __size_func, __set_op, &__buf]() {
-        auto __buffer = __buf.get();
+        auto __buf_mem = __buf.get();
```

Comment:
I would either prefer to do not touch this file at all or to find a better name for a variable.

For example `__tmp_buffer`, or `__tmp_memory` or something else.

Reply:
Fixed: renamed to `__tmp_memory`